data.sql

```diff
@@ -61,3 +61,50 @@ UPDATE animal SET owner_id = 4 WHERE name = 'Blossom';
 UPDATE animal SET owner_id = 5 WHERE name = 'Angemon';
 
 UPDATE animal SET owner_id = 5 WHERE name = 'Boarmon';
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('William Tatcher', 45, date '2000-04-23');
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('Maisy Smith', 26, date '2019-01-17');
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('Stephanie Mendez', 64, date '1981-05-04');
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('Jack Harkness', 38, date '2008-06-08');
+
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=Insert%20the%20following%20data%20for%20specialties,Jack%20Harkness%20is%20specialized%20in%20Digimon.), your are supposed to insert data into the Specialization join table. Kindly add the insertion data into this data.sql file

Reply:
@GedeonTS , I forgot to add the SQL queries to the data.sql file. Thank you for letting me know. I will do that immediately.